backend/test/backend_tests/rpc_file_test.clj

```diff
@@ -1658,3 +1658,174 @@
             components (get-in result [:data :components])]
         (t/is (not (contains? components c-id)))))))
 
+
```

Comment:
you can remove all this blankspaces

Reply:
This blank lines are explicit, mainly because the test file is large and the test case is also large, so when you scroll on file, you have a better visual feedback for where a test case impl starts and ends.